spec/safe_statements_spec.rb

```diff
@@ -1841,7 +2036,7 @@ def up
             end.not_to raise_error
           end
 
-          describe "when not configured to disallow two-step new column and adding default" do
+          context "when not configured to disallow two-step new column and adding default" do
```

Comment:
Disapproving stare.

Reply:
😆  Are you in the camp of just using `describe` everywhere? Really don't feel strongly about this either so I can change it back

Reply:
Yeah, I use `describe` everywhere for simplicity. I find the describe/context thing a significant bike-shed debate with no clear boundaries.

Reply:
Done in https://github.com/braintree/pg_ha_migrations/pull/113/commits/4815e0476f6d3d3815ac5365218db26d4b4aacea